GameServerLib/Config.cs

```diff
@@ -90,6 +97,30 @@ private void LoadConfig(Game game, string json)
             MapData = ContentManager.GetMapData(GameConfig.Map);
             MapSpawns = ContentManager.GetMapSpawns(GameConfig.Map);
         }
+
+        private string getContentPath()
```

Comment:
Start the first letter of a function with a capital letter in order to comply with our code style guidelines. (See https://github.com/LeagueSandbox/GameServer/blob/indev/CONTRIBUTING.md)